docs/learn/learn-DOT-KSM-bridge.md

```diff
@@ -0,0 +1,75 @@
+---
+id: learn-dot-ksm-bridge
+title: Polkadot <> Kusama Bridge
+sidebar_label: DOT <> KSM Bridge
+description: Overview of Polkadot and Kusama Bridge.
+keywords: [Bridge, XCM, Bridge Hub]
+slug: ../learn-dot-ksm-bridge
+---
+
+Both Polkadot and Kusama blockchain networks achieve finality through GRANDPA consensus, which
+enables trustless bridging of both the networks through their respective Bridge Hubs. Polkadot
+Bridge Hub runs a [light client of Kusama network](https://polkadot.polkassembly.io/referenda/545)
+and Kusama Bridge Hub runs a
+[light client of Polkadot network](https://kusama.polkassembly.io/referenda/354), which were both
+enabled through their respective OpenGov referenda. This trustless bridge allows Polkadot Asset Hub
+accounts to hold wrapped KSM tokens and Kusama Asset Hub accounts to hold wrapped DOT tokens.
```

Comment:
This somewhat reads that the bridged DOT and KSM are only available/usable on asset hubs. Let's rephrase. Maybe something like:

```suggestion
enabled through their respective OpenGov referenda. This trustless bridge allows Polkadot Asset Hub
to bridge in wrapped KSM tokens and Kusama Asset Hub to bridge in wrapped DOT tokens, thus making
DOT available to all Kusama parachains and KSM to all Polkadot parachains.
```